htdocs/class/xoopsform/formdhtmltextarea.php

```diff
@@ -149,19 +149,34 @@ public function render()
         // fonts
         $ret .= $this->fontArray();
         // length checker
-        $ret .= "<input type='button' onclick=\"XoopsCheckLength('" . $this->getName() . "', '" . @$this->configs['maxlength'] . "', '" . _XOOPS_FORM_ALT_LENGTH . "', '" . _XOOPS_FORM_ALT_LENGTH_MAX . "');\" value=' ? ' title='" . _XOOPS_FORM_ALT_CHECKLENGTH . "' />";
+        if ($GLOBALS['xoopsConfig']['bootstrap'] == true){
+            $ret .= "<button type='button' class='btn btn-default btn-sm' onclick=\"XoopsCheckLength('" . $this->getName() . "', '" . @$this->configs['maxlength'] . "', '" . _XOOPS_FORM_ALT_LENGTH . "', '" . _XOOPS_FORM_ALT_LENGTH_MAX . "');\" title='" . _XOOPS_FORM_ALT_CHECKLENGTH . "'><span class='glyphicon glyphicon-question-sign' aria-hidden='true'></span></button>&nbsp;";
+            $ret .= "</div></div>";
+        } else {
+            $ret .= "<input type='button' onclick=\"XoopsCheckLength('" . $this->getName() . "', '" . @$this->configs['maxlength'] . "', '" . _XOOPS_FORM_ALT_LENGTH . "', '" . _XOOPS_FORM_ALT_LENGTH_MAX . "');\" value=' ? ' title='" . _XOOPS_FORM_ALT_CHECKLENGTH . "' />";
+        }
         $ret .= "<br>\n";
         // the textarea box
-        $ret .= "<textarea id='" . $this->getName() . "' name='" . $this->getName() . "' title='" . $this->getTitle() . "' onselect=\"xoopsSavePosition('" . $this->getName() . "');\" onclick=\"xoopsSavePosition('" . $this->getName() . "');\" onkeyup=\"xoopsSavePosition('" . $this->getName() . "');\" cols='" . $this->getCols() . "' rows='" . $this->getRows() . "'" . $this->getExtra() . '>' . $this->getValue() . "</textarea><br>\n";
+        if ($GLOBALS['xoopsConfig']['bootstrap'] == true){
+            $ret .= "<div class='form-group form-inline'><textarea class='form-control' id='" . $this->getName() . "' name='" . $this->getName() . "' title='" . $this->getTitle() . "' onselect=\"xoopsSavePosition('" . $this->getName() . "');\" onclick=\"xoopsSavePosition('" . $this->getName() . "');\" onkeyup=\"xoopsSavePosition('" . $this->getName() . "');\" cols='" . $this->getCols() . "' rows='" . $this->getRows() . "'" . $this->getExtra() . '>' . $this->getValue() . "</textarea></div>\n";
+        } else {
```

Comment:
__[phpmd]__
* The method render uses an else expression. Else is never necessary and you can simplify the code to work without else.

<!-- Comment by [SideCI](https://sideci.com) -->